gridmap_generator:
  ros__parameters:
    resolution: 1.0 # float32: meters/grid_cell
    grid_width: 100 # uint32: num_cols of gridmap
    grid_height: 100 # uint32: num_rows of gridmap
    update_delay: 1.0 # float32: How long between map updates
    z_filter_height: 0.1 # float32: How many meters below boat before points filtered out
